File: mario-py/mario.py
from cs50 import get_int
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
for row in range(1 , height + 1):
    for space in range(0 , height - row):
        print(" " , end="")
    for dash in range(0 , row):
        print("#", end="")
    print(end="  ")
    for dash in range(0 , row):
        print("#", end="")
    print()

File: mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (int row = 1; row <= height; row++)
    {
        //spaces for 1st wedge
        for (int space = 0; space < height - row; space++)
        {
            printf(" ");
        }

        //dashes for 1st wedge
        for (int dash = 0; dash < row; dash++)
        {
            printf("#");
        }

        //gaps
        printf("  ");

        //dashes for 2nd wedge
        for (int dash = 0; dash < row; dash++)
        {
            printf("#");
        }
        printf("\n");
    }
}